.changeset/lemon-mails-smoke.md

```diff
@@ -0,0 +1,19 @@
+---
+'xstate': major
+---
+
+Observing an actor via `actor.subscribe(...)` no longer immediately receives the current snapshot. Instead, the current snapshot can be read from `actor.getSnapshot()`, and observers will receive snapshots only when a transition in the actor occurs.
```

Comment:
note for me - this might require a small tweak in the wording if we decide to go with https://github.com/statelyai/xstate/pull/3912